core/pom.xml

```diff
@@ -38,6 +38,10 @@
         <download-maven-plugin.version>1.4.2</download-maven-plugin.version>
         <protobuf-maven-plugin.version>0.6.1</protobuf-maven-plugin.version>
         <maven-compiler-plugin.verion>3.8.1</maven-compiler-plugin.verion>
+        <akka.version>2.5.31</akka.version>
+        <akka.http.version>10.1.8</akka.http.version>
+        <scala.binary.version>2.12</scala.binary.version>
```

Comment:
All of these should not be re-introduced in the module pom file. They should automatically get picked up from the parent pom file. See other examples for reference.